src/main/kotlin/com/example/minibank/services/impl/FTServiceImpl.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.minibank.services.impl
+
+import com.example.minibank.config.FTProperties
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end